validator/coordinator/src/main/java/tech/pegasys/teku/validator/coordinator/ValidatorApiHandler.java

```diff
@@ -383,12 +383,13 @@ public void sendSignedAttestation(
         .onAttestation(ValidateableAttestation.fromValidator(attestation))
         .finish(
             result -> {
-              result.ifInvalid(
-                  reason ->
-                      VALIDATOR_LOGGER.producedInvalidAttestation(
-                          attestation.getData().getSlot(), reason));
-              dutyMetrics.onAttestationPublished(attestation.getData().getSlot());
-              performanceTracker.saveProducedAttestation(attestation);
+              if (!result.isInvalid()) {
```

Comment:
(nit) m.b. `isValid()` would be more readable ?

Reply:
It's an odd case where it strictly is checking for not invalid but it may not be confirmed as valid yet - could be saved for later for example.